README.md

```diff
@@ -1,16 +1,51 @@
-# Release Integration Tests
+# Release Integration Test
 
+This README provides instructions on running a test using the provided shell script.
 
-## 1. FBC Test (happy path):
-The purpose of this release test (FBC) is to verify the functionality and stability of feature in RHTAP.
-Test is a "happy-path" test and it's running on the internal RHTAP stage cluster.
+## Test Steps and Expected Result
+
+1. **Pre-Running Test:** The following setup actions need to be completed before running the test script.
+   The test is running against RHTAP-Stage, so a user wants to monitior the test running throught UI or CLI 
+   will need an access to the cluster and be joined to the workspaces *release-dev-team* and *release-managed-team*. 
+   
+   Our test expect to eun on both workspaces above, and it's expected (based on the fllow agreed on), namespaces dev and managed exist, ploicy, service account, secrets already deployed on those two dedicated workspaces 
+   *release-dev-team* and *release-managed-team*.
+
+   Required that a user to run the test when connecting the RHTAP if the user is not logged in to SSO, so a browser page will pop-up and require an SSO login. 
```

Comment:
This reads weird
It is required that the user running the test is connected to RHTAP. If the user is not logged into SSO, a browser page will pop-up and ask for an SSO login.